src/app/background.ts

```diff
@@ -239,8 +239,8 @@ function isJSONGraphQLBody(details: chrome.webRequest.WebRequestBodyDetails) {
     if (!details.requestBody?.raw?.[0]?.bytes) return false;
 
     const body = enc
-      .decode(details.requestBody.raw[0].bytes)
-      .replace(/\s/g, ""); 
+      .decode(details.requestBody.raw[0].bytes) ;
+    const stringified = body.replace(/\s/g, "")
```

Comment:
Your prettier didn't work here 😛 
```suggestion
    const stringified = body.replace(/\s/g, "");
```